webhook/forward.go

```diff
@@ -35,16 +35,16 @@ func NewCmdForward(runF func(*hookOptions) error) *cobra.Command {
 		Out: os.Stdout,
 	}
 	cmd := &cobra.Command{
-		Use:   "forward --events=<event_types> --repo|org=<repo|org> [--port=<port>] [--host=<host>]",
+		Use:   "forward --events=<event_types> --repo|org=<repo|org> [--host=<host>] [--url=\"<url>\"]",
 		Short: "Receive test events on a server running locally",
 		Long: heredoc.Doc(`To output event payloads to stdout instead of sending to a server,
-			omit the --port flag. If the --host flag is not specified, webhooks will be created against github.com`),
+			omit the --url flag. If the --host flag is not specified, webhooks will be created against github.com`),
```

Comment:
I think at this point we can totally remove the "If the --host flag is not specified..." sentence because it's really not for customers but for us developers of the extension.

Reply:
I think we should probably keep it as it is in the `Use` declaration. Otherwise, we could make it completely undocumented. What do you think?

Reply:
I'm just going to remove it here actually. Think it's fine to have it in the option-by-option docs further below.